agent/php_execute.c

```diff
@@ -1035,7 +1035,7 @@ static void nr_php_execute_metadata_add_code_level_metrics(
 
 #undef CHK_CLM_STRLEN
 
-  lineno = nr_php_zend_execute_data_lineno(execute_data);
+  lineno = nr_php_zend_function_lineno(execute_data->func);
```

Comment:
Is this line needed? Isn't `lineno` set either on line 1049 or line 1054?

Reply:
I refactored this quite a bit with the latest PR I'm working on.  I've changed this function to populate the observer metadata and moved all the CLM logic into the php_txn function.
The new PR will do the following regarding this logic:
1. Renamed nr_php_execute_metadata_add_code_level_metrics to  nr_php_observer_metadata_init as this will be the way we populate the metadata.  We don't need to do duplicate effort with nr_php_execute_metadata_init
anymore.

2. Since nr_php_execute_metadata_init
 Is now for populating metadata, moved all CLM checks out of nr_php_observer_metadata_init
 And into nr_php_txn_add_code_level_metrics
 Where it is more appropriate.